### DISCUSSION TITLE
package name has a new keyword

### DISCUSSION THREAD

**Discussion by Gokulv617** [airsdk/Adobe-Runtime-Support#2847] [Q&A]
Hi ,
Previously, we were using the Airsdk version 'AIR_SDK_33.1.1.795" version, which was working fine. Now, we are trying to use the Airsdk version 'AIR_SDK_50.2.3.4" for Android 33. I have encountered an error while trying to create an apk or aab file. I have discovered that my package name has a new keyword. I require your assistance in fixing this error.
![error](https://github.com/airsdk/Adobe-Runtime-Support/assets/43432782/4d804bd2-5f56-4493-b072-23172658fc89)

**Comment by itlancer** [comment]
@Gokulv617 
It's reserved word. Full answer here: https://github.com/airsdk/Adobe-Runtime-Support/issues/2408#issuecomment-1375794403
So I think you should change app ID to resolve this issue.

**Reply by Gokulv617** [reply]
Okay, Thank you for your prompt response.
